src/component/spinner.js

```diff
@@ -84,6 +129,7 @@ const resizeableStyles = StyleSheet.create({
 });
 
 export const ResizeableSpinner = ({
+  continuous,
```

Comment:
`continuous` needed to be passed into `SpinnerProgress`. Since `ResizableSpinner` sits between the top level spinner and the `SpinnerProgress` components, it has to go through `ResizableSpinner`